compiler-rt/lib/builtins/cpu_model.c

```diff
@@ -1363,6 +1364,8 @@ static void __init_cpu_features_constructor(unsigned long hwcap,
     setCPUFeature(FEAT_SME_I64);
   if (hwcap2 & HWCAP2_SME_F64F64)
     setCPUFeature(FEAT_SME_F64);
+  if (hwcap2 & HWCAP2_SME_FA64)
```

Comment:
Does HWCAP2_SME_FA64 need to be defined above? For example, the other SME features are defined around line 1127:
```
#ifndef HWCAP2_SME
#define HWCAP2_SME (1 << 23)
#endif
#ifndef HWCAP2_SME_I16I64
#define HWCAP2_SME_I16I64 (1 << 24)
#endif
```

Reply:
I've defined it now.